### PR TITLE
changed units for throughput #7250

### PR DESCRIPTION
**Pull request by mitjaziv**
changed units for throughput for [Feature request] #7250

### CODE REVIEW

File: public/app/core/utils/kbn.js [hunk 1]
```diff
@@ -427,13 +427,13 @@ function($, _) {
...
-  kbn.valueFormats.wps  = kbn.formatBuilders.simpleCountUnit('wps');
-  kbn.valueFormats.iops = kbn.formatBuilders.simpleCountUnit('iops');
-  kbn.valueFormats.opm = kbn.formatBuilders.simpleCountUnit('opm');
-  kbn.valueFormats.rpm = kbn.formatBuilders.simpleCountUnit('rpm');
-  kbn.valueFormats.wpm = kbn.formatBuilders.simpleCountUnit('wpm');
+  kbn.valueFormats.opssec    = kbn.formatBuilders.simpleCountUnit('ops/s');
+  kbn.valueFormats.readsec   = kbn.formatBuilders.simpleCountUnit('reads/s');
+  kbn.valueFormats.writesec  = kbn.formatBuilders.simpleCountUnit('writes/s');
+  kbn.valueFormats.ioopssec  = kbn.formatBuilders.simpleCountUnit('I/O ops/s');
+  kbn.valueFormats.opsmin    = kbn.formatBuilders.simpleCountUnit('ops/m');
```
Comment (EdSchouten) [inline public/app/core/utils/kbn.js]: 'm' is the abbreviated form of metres.

Comment (mitjaziv) [inline public/app/core/utils/kbn.js]: You are correct, i will change to minutes.

**Comment by torkelo** [comment]
This would cause massive breaking changes in existing panels so will have to close it. 

Most of these units are more commonly used in the graph ticks where there is limited space for long unit names.

**Comment by EdSchouten** [comment]
If this can't be integrated, could we at least introduce a textbox where we can fill in the desired unit manually?

**Comment by torkelo** [comment]
Yes, should be possible

**Pull request closed without merging**